hazelcast/src/main/java/com/hazelcast/client/console/ClientConsoleApp.java

```diff
@@ -117,30 +119,51 @@
 
     private final LineReader lineReader;
     private final PrintWriter writer;
-    private volatile boolean running;
-
     private final HazelcastInstance client;
 
+    private volatile boolean running;
+
     public ClientConsoleApp(@Nonnull HazelcastInstance client) {
-       this(client, null);
+        this(client, null);
     }
 
     public ClientConsoleApp(@Nonnull HazelcastInstance client, @Nullable PrintWriter writer) {
         this.client = client;
+
+        Terminal terminal = createTerminal();
```

Comment:
Is it closed properly? The `Terminal.terminal()` javadoc says

>Terminals should be closed properly using the `Terminal.close()` method in order to restore the original terminal state.

Reply:
Ok also add a fix to close it: https://github.com/hazelcast/hazelcast/pull/20973/commits/91b7279ce1d0173f2335f1361a485f5133856050